File: bronze4/31608.cpp
#include <iostream>
#include <string>

int main(void) 
{
    int n = 0;
    std::cin >> n;

    std::string a, b;
    std::cin >> a >> b;

    int count = 0;
    for (int i = 0; i < n; i++) 
    {
        if (a[i] != b[i]) count++;
    }

    std::cout << count;
    
    return 0;
}

File: bronze4/31608.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let input = [];
rl.on('line', (line) => {
    input.push(line.trim());
    if (input.length === 3) {
        const n = parseInt(input[0]);
        const a = input[1];
        const b = input[2];
        
        let count = 0;
        for (let i = 0; i < n; i++) {
            if (a[i] !== b[i]) count++;
        }

        console.log(count);
        rl.close();
    }
});
